README.md

```diff
@@ -23,40 +54,41 @@ cd js-rln/example
 npm install  # or yarn
 
 npm start
-
 ```
 
 Browse http://localhost:8080 and open the dev tools console to see the proof being generated and its verification
 
-### Usage
+## Usage
 
-#### Initializing the library
+### Initializing the Library
 
 ```js
 import * as rln from "@waku/rln";
 
-const rlnInstance = wait rln.create();
+const rlnInstance = await rln.create();
 ```
 
-#### Generating RLN membership keypair
```

Comment:
Maybe worth it to mention that rln credentials are compatible with semaphore credentials